docs/howto/openstack/nova/resize-server.md

```diff
@@ -0,0 +1,125 @@
+# Resizing a server
+
+This guide will walk you through the required steps to change the number of CPU cores and the amount of memory your server has access to, this is done my changing the server's [flavor](/reference/flavors/).
+
+[Resize](https://docs.openstack.org/nova/latest/admin/configuration/resize.html) (or Server resize) is the ability to change the flavor of a server, thus allowing it to upscale or downscale according to user needs. A resize operation is a two-step process for the user: the first step is to perform the resize, and the second step is to either confirm (verify) success and release the old server or to declare a revert to release the new server and restart the old one.
+
+## Prerequisites
+
+You need to have a server you wish to resize. Additionally, if you prefer to work with the OpenStack CLI, then make sure to properly [enable it first](/howto/getting-started/enable-openstack-cli/).
```

Comment:
I find enabling CLI as super weird statement. Can we re-phrase it to create CLI user or smth as we don't disable CLI/API - we just don't provide access details on account creation.

Reply:
I agree, however, this is the exact same language as used in many other guides and also in the referenced "getting started" guide.

Reply:
"Enabling" means "making something work". In order to make the `openstack` CLI work, you need a user, a password, and properly configured environment variables. Without the foregoing, it is useless. So, the wording is fine.